src/server/game/World/World.cpp

```diff
@@ -1124,14 +1124,9 @@ void World::LoadConfigSettings(bool reload)
         m_int_configs[CONFIG_BATTLEGROUND_REPORT_AFK] = 3;
     }
     m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]        = sConfigMgr->GetIntDefault("Battleground.RespawnTime", 30);
-    if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] < 3)
+    if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] < 3 || m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] > 120)
     {
-        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
-        m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] = 30;
-    }
-    else if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] > 120)
-    {
-        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
+        sLog->outError("Battleground.RespawnTime (%i) must be in range 3 - 120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
```

Comment:
```suggestion
        sLog->outError("Battleground.RespawnTime (%i) must be in range 3 - 120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]);
```

Reply:
"3-120" and "3..120" matches the styling in the rest of the core